src/lib/dataElements/runtimeEnvironment.js

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+'use strict';
+
+/**
+ * The launch environment data element.
+ * @param {Object} settings The data element settings object.
+ * @param {string} settings.attribute The attribute that should be returned.
+ * @returns {string}
+ */
+
+var window = require('@adobe/reactor-window');
+
+module.exports = function (settings, event) {
+  switch (settings.attribute) {
+    case 'buildDate':
+      return turbine.buildInfo.buildDate;
+    case 'environment':
+      return turbine.buildInfo.environment;
```

Comment:
faux pas. Use `container.environment.stage`

Reply:
They don't have access to container. I updated the code to read if from the _satellite variable.